src/governance/governance.cpp

```diff
@@ -140,8 +202,16 @@ bool CGovernanceManager::SerializeVoteForHash(const uint256& nHash, CDataStream&
     return cmapVoteToObject.Get(nHash, pGovobj) && pGovobj->GetVoteFile().SerializeVoteToStream(nHash, ss);
 }
 
-MessageProcessingResult CGovernanceManager::ProcessMessage(CNode& peer, CConnman& connman, PeerManager& peerman, std::string_view msg_type, CDataStream& vRecv)
+void CGovernanceManager::AddPostponedObject(const CGovernanceObject& govobj)
 {
+    LOCK(cs);
+    mapPostponedObjects.insert(std::make_pair(govobj.GetHash(), govobj));
```

Comment:
@coderabbitai, should we use emplace here to avoid an extra copy due to constructing then inserting?

Reply:
Resolved in latest push.